docs/guide/snapshot.md

```diff
@@ -98,7 +98,7 @@ It will compare with the content of `./test/basic.output.html`. And can be writt
 
 ## Image Snapshots
 
-It's also possible to snapshot images using [`jest-image-snapshot`](https://github.com/americanexpress/jest-image-snapshot).
+It's also possible to snapshot images using [`jest-image-snapshot`](https://github.com/americanexpress/jest-image-snapshot). When using Typescript, you will need to [extend the `Assertion` interface](/guide/extending-matchers.html).
```

Comment:
This doesn't really explain anything. If you need to have types for this package, you just need to have `@types/jest-image-snapshot` installed. This is something that should be explained by the package itself

Reply:
`jest-image-snapshot` users need to use `expect.extend`, which unfortunately doesn't add typing. However, going over their docs again, there is [a section](https://github.com/americanexpress/jest-image-snapshot#usage-in-typescript) mentioning the necessity to add a matcher. I will close the PR as the package docs (partially) explain how to do it.